### ISSUE TITLE
Create option to prepend template

### ISSUE THREAD

**Issue by chrisabrams** [littlstar/starplate#6]
I'm still trying to wrap my head around the project, but I believe this is where the template is actually being added to the view? https://github.com/littlstar/starplate/blob/master/src/view.js#L208

If so, could it be possible to make an option to prepend?

Maybe `render` gets a second argument, with optional options? Such as:

```
render(document.body, {placement: 'prepend'})
```

**Comment by jwerle** [comment]
Yeah I think that would be great. I'm not at my computer right now, but
we'd happily accept a PR!

**Comment by chrisabrams** [comment]
Me either :O but expect a PR soon!

**Comment by jwerle** [comment]
@chrisabrams Cool, I've added you as a collaborator to this project !
